src/MSBuild.UnitTests/CommandLineSwitches_Tests.cs

```diff
@@ -1055,6 +1055,43 @@ public void AppendParameterizedSwitchesTests2()
             Assert.Equal("build", parameters[2]);
         }
 
+        /// <summary>
+        /// Verifies that the Target property is unquoted and parsed properly.
+        /// This will remove the possibility to have the ';' in the target name. 
+        /// </summary>
+        [Theory]
+        [InlineData("/t:Clean;Build", "\"Clean;Build\"")]
```

Comment:
Should this be

```suggestion
        [InlineData("/t:\"Clean;Build\"", "\"Clean;Build\"")]
```

? If not, why not?

Reply:
Ok, this sounds good then. I don't think a follow-up refactor is necessary but I don't object to it either.